apps/web/cypress/tests/notifications-editor.spec.ts

```diff
@@ -120,6 +120,51 @@ describe('Notifications Creator', function () {
       cy.getByTestId('trigger-snippet-btn').click();
       cy.location('pathname').should('equal', '/templates');
     });
+    it('should create multiline in-app notification, send it and receive', function () {
```

Comment:
create an in-app notification with multiline content, trigger it and then verify if it's rendered correctly on the notifications list

Reply:
A great idea for a test!